nbfetch/__init__.py

```diff
@@ -1,12 +1,13 @@
 from .version import __version__
 from .handlers import HSLoginHandler, SyncHandler, HSyncHandler, UIHandler, HSHandler
-from notebook.utils import url_path_join
+from jupyter_server.utils import url_path_join
 from tornado.web import StaticFileHandler
 import os
 
 import jupyter_server
 from jupyter_server.extension.application import ExtensionApp
-from notebook import DEFAULT_STATIC_FILES_PATH, DEFAULT_TEMPLATE_PATH_LIST
+from jupyter_server import DEFAULT_TEMPLATE_PATH_LIST
+from jupyter_server import DEFAULT_STATIC_FILES_PATH
```

Comment:
When testing locally it looks like there are issues with the static files
<img width="1178" alt="image" src="https://github.com/user-attachments/assets/6c9ef27d-44fc-45f0-8055-4240043baefc" />